rpc/core/routes.go

```diff
@@ -29,7 +29,7 @@ var Routes = map[string]*rpc.RPCFunc{
 	"broadcast_tx_async":  rpc.NewRPCFunc(BroadcastTxAsync, "tx"),
 
 	// abci API
-	"abci_query": rpc.NewRPCFunc(ABCIQuery, "path,data,prove"),
+	"abci_query": rpc.NewRPCFunc(ABCIQuery, "path,data,height,prove"),
```

Comment:
this is definitely the simplest approach, but I believe @cloudhead was arguing to remove the bool from here and use a separate `abci_query_trusted` method for prove = false, so the default is it's always true. Would that be better ? It might be nice if the options struct was an arg here that could go in through the jsonrpc, but itd be a total pain to call it using the URI - maybe we just shouldn't worry too much about that usecase, I kind of see us eventually deprecating the URI interface eventually due to issues like this

Reply:
> be a total pain to call it using the URI

yeah.

As I understood, @ethanfrey and @cloudhead don't care as long as the HttpClient (or LocalClient) defaults to prove=true. And that's what I did: our both clients have this interface

```
ABCIQuery(path string, data []byte) { return ABCIQueryWithOptions(path, data, abci.DefaultQueryOptions) }
ABCIQueryWithOptions(path string, data []byte, opts abci.QueryOptions) { ... }
```

Or this is counterintuitive? I mean when the rpc core method defaults to prove=false, but the official client defaults to prove=true?

Reply:
I was more concerned with the Go API - but I do think they should all default to prove=true/trusted=false

Reply:
"yeah, it will be much easier if we rename prove to trusted"